doc/reference_guide/source/conf.py

```diff
@@ -21,13 +21,13 @@
 # -- Project information -----------------------------------------------------
 
 project = 'Reference Guide'
-copyright = '2019, R. W. Ford, J. Henrichs, I. Kavcic, A. R. Porter, S. Siso'
+copyright = '2019-2020, R. W. Ford, J. Henrichs, I. Kavcic, A. R. Porter, S. Siso'
```

Comment:
pycodestyle says line is too long. I'll not ask for pylint compliance for the file seeing as this is a small modification.

Reply:
Oh yes, fixed.